spec/mem/atomic.md

```diff
@@ -18,6 +18,10 @@ pub enum Atomicity {
 
     /// A non-atomic memory access.
     None,
+
+    /// An access that can not be seen by any other thread except
+    /// for the one executing it. Requires comment explaining reason if used.
```

Comment:
This comment is wrong,. when spawning a thread then it is a different thread seeing the access!
```suggestion
    /// The initial write to an allocation that is only accessible by a single thread.
    /// Will be ignored for data race purposes. Requires comment explaining reason if used.
```

Reply:
This is not needed any more since we will get rid of Init.